### PR TITLE
Completed css-positioning - Lisa Nguyen

### PR DESCRIPTION
**Pull request by lisa-t-nguyen**
chrome_hBMnMVOE0r
chrome_sFdY6ByLoz

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
```
Comment (Zhiwutian) [inline css-positioning/absolute.css:7]: you want to make sure to have `position:relative` set on this element

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 162px;
+  width: 300px;
+}
+
+.overlay {
```
Comment (Zhiwutian) [inline css-positioning/absolute.css:36]: position:absolute should be placed on this element

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good Lisa, just a couple small updates!

File: css-positioning/absolute.css (hunk 1 above)
Comment (lisa-t-nguyen) [reply css-positioning/absolute.css:7]: Oops! I accidentally placed that on .meme > img instead! I shall fix that!

**Comment by lisa-t-nguyen** [comment]
image

Updated screenshot for absolute.html

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**